openapi.yaml

```diff
@@ -30043,6 +30436,132 @@ components:
           x-linode-cli-display: 6
           example: "2018-01-01T13:46:32"
           readOnly: true
+    PlacementGroup:
+      type: object
+      properties:
+        id:
+          type: integer
+          nullable: false
+          description: >
+            The placement group's ID. You need to provide it for all operations impacting it.
+          example: 528
+          x-linode-cli-display: 1
+        label:
+          type: string
+          x-linode-cli-display: 1
+          x-linode-filterable: true
+          description: |
+            The unique name set for the placement group. A label has these constraints:
+
+            * It needs to begin and end with an alphanumeric character.
+            * It can only consist of alphanumeric characters, hyphens (`-`), underscores (`_`) or periods (`.`).
+          example: PG_Miami_failover
+          minLength: 1
+        region:
+          type: string
+          x-linode-cli-display: 1
+          x-linode-filterable: true
+          readOnly: true
+          description: >
+            The [region](/docs/api/regions/#regions-list) where the placement group was deployed.
+          example: us-mia
+        affinity_type:
+          type: string
+          x-linode-filterable: true
```

Comment:
Would it make sense to add an `x-linode-cli-display: 1` here to make sure the affinity type is displayed by default in the CLI output?